src/main/java/net/snowflake/ingest/streaming/internal/ChannelMetadata.java

```diff
@@ -17,6 +17,7 @@ class ChannelMetadata {
   private final Long clientSequencer;
   private final Long rowSequencer;
   @Nullable private final String offsetToken;
```

Comment:
```suggestion
  @Nullable private final String endOffsetToken;
```

this is basically an internal field name, does it make sense to rename it as well for better readability?